plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -692,6 +692,10 @@
                 {
                     "type": "cwsprChatHasProjectContext",
                     "required": false
+                },
```

Comment:
Is this metric different in VSC? Is it possible to move this to common?

Reply:
Yes, we can add this to common: https://github.com/aws/aws-toolkit-common/pull/883
This was used in multiple events and declared in IDE instead of common so I went with traditional approach.